### PR TITLE
feature/1920 — Add Salt Lake City Int'l (SLC)

### LINKED ISSUE #1920: Add SLC - Salt Lake City Int'l
Add SLC.

### PR DESCRIPTION
**Pull request by MichaelDMcCracken**
Resolves #1920

The purpose of this pull request is to add SLC.

### CODE REVIEW

File: assets/airports/kslc.json [hunk 1]
```diff
@@ -0,0 +1,9278 @@
...
+            "draw": [
+                ["HOPTO", "ZIONZ", "KOOGR", "ROMMN", "KIMMR*"],
+                ["ROMNN", "PLEZZ", "EYELO*"],
+                ["ZIONZ", "GITLN", "LODUY", "KROST*"],
+                ["LODUY", "URNUW", "BCE*"],
+                ["URNUW", "EHK*"]
+            ]
+        }
+    },
+    "stars": {
```
Comment (Fechulo) [inline assets/airports/kslc.json:568]: Missing Brigham City Four (LHO4) Arrival

File: assets/airports/airportLoadList.json [hunk 2]
```diff
@@ -337,6 +337,12 @@
         "name": "San Francisco International Airport",
         "premium": false
     },
+    {
+        "icao": "kslc",
+        "level": "hard",
```
Comment (Fechulo) [inline assets/airports/airportLoadList.json:342]: With 53.6acph, this should be medium

**Review by Fechulo — CHANGES_REQUESTED** [review]
Sorry for the long wait, a few small changes needed

File: assets/airports/kslc.json [hunk 3]
```diff
@@ -0,0 +1,9278 @@
...
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 160,
+        "speed": 10
+    },
+    "arrivalRunway": "16L",
+    "departureRunway": "16R",
+    "defaultMaps": ["South Flow", "ZLC Perimeter"],
+    "airspace": [
```
[the remark this answers is not in the document]
Comment (MichaelDMcCracken) [reply assets/airports/kslc.json:27]: The ZLC VATSIM documentation I had was very generic, didn't have a full-on airspace detail like some other ARTCCs.

File: assets/airports/kslc.json (hunk 1 above)
Comment (MichaelDMcCracken) [reply assets/airports/kslc.json:568]: fixed

File: assets/airports/kslc.json [hunk 4]
```diff
@@ -0,0 +1,9278 @@
...
+            "draw": [
+                ["MLF", "BEVRR", "DTA", "SLINA", "BCE"],
+                ["DTA", "JAMMN", "DRAPR", "SPIEK", "HEIRY", "PITTT", "MAGNE", "QUIPA", "RROYY"]
+            ]
+        },
+        "JAMMN5": {
+            "icao": "JAMMN5",
+            "name": "Jammin Five",
+            "entryPoints": {
+                "BCE": ["BCE", "SLINA", "DTA"],
```
[the remark this answers is not in the document]
Comment (MichaelDMcCracken) [reply assets/airports/kslc.json:638]: no idea why I did that, fixed.

File: assets/airports/kslc.json [hunk 5]
```diff
@@ -0,0 +1,9278 @@
...
+                "KSLC14": [],
+                "KSLC16L": [],
+                "KSLC16R": [],
+                "KSLC17": [],
+                "KSLC32": [],
+                "KSLC34L": [],
+                "KSLC34R": [],
+                "KSLC35": []
+            },
+            "body": [["CORVR", "A130+"], ["CGULL", "A150+"], "MOFER", "SCOVL", "DRYAD", "TWF"],
```
[the remark this answers is not in the document]
Comment (MichaelDMcCracken) [reply assets/airports/kslc.json:409]: FIXED

File: assets/airports/kslc.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Fechulo) [reply assets/airports/kslc.json:27]: Yeah I couldn't find much of anything

**Review by Fechulo — APPROVED** [review]

**Pull request merged**